include/git2/odb.h

```diff
@@ -171,6 +173,40 @@ GIT_EXTERN(int) git_odb_exists(git_odb *db, const git_oid *id);
 GIT_EXTERN(int) git_odb_exists_prefix(
 	git_oid *out, git_odb *db, const git_oid *short_id, size_t len);
 
+typedef struct git_odb_expand_id {
+	/** The object ID to expand */
+	git_oid id;
+
+	/** The length of the object ID */
+	unsigned short length;
```

Comment:
Hah, like that time thousands the OID lookups in production became ambiguous because we were passing bytes instead of nibbles. :))))))

...good times.